docs/api/components/schemas.yaml

```
@@ -2042,7 +2042,76 @@ Startup-config:
           type: string
           default: 10s
         server:
-          description: Couchbase Server connection string/URL.
+          description: |
+            Couchbase Server connection string/URL for bootstrap configuration.
+            
+            The connection string should only reference Couchbase Server Data (KV) nodes. 
+            Using other node types (Query, Index, Analytics, or Search nodes) may cause 
+            connection failures and unreliable operation.
+            
+            **Connection String Format**
+            Sync Gateway supports the ability to specify multiple hosts in the configuration. 
+            Sync Gateway supports both the couchbase:// and http:// schemes for specifying 
```

```suggestion
            Sync Gateway supports both the couchbase:// and couchbases:// schemes for specifying 
```

https://docs.couchbase.com/go-sdk/current/howtos/managing-connections.html#connection-strings